scripts/verify-docs.py

```diff
@@ -35,7 +35,7 @@
 examples_directory = 'examples'
 functions_directory = 'functions'
 lang_dirs = ['go', 'ts']
-examples_directories_to_skip = ['_template']
+examples_directories_to_skip = ['_template', 'render-helm-chart-kustomize-private-oci', 'render-helm-chart-kustomize-private-git']
```

Comment:
Please don't forget to remove this change in the other PR.

Reply:
Left a comment to remind myself https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/875#issuecomment-1149289269